bin/npm-force-upgrade.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+var path = require('path')
+var join = path.join
+var resolve = path.resolve
+var read = require('fs').readFileSync
+var exec = require('child_process').execSync
+var args = process.argv.slice(2)
+var argv = require('minimist')(args, {
+  alias: {
+    h: 'help',
+    v: 'version',
+    p: 'package'
+  },
+  default: {
+    package: 'package.json'
+  }
+})
+
+if (argv.help) {
+  var path = join(__dirname, 'help.txt')
+  var help = read(path, 'utf8')
+  console.log(help)
+  process.exit()
+}
+
+if (argv.version) {
+  var path = join(__dirname, '../package.json')
+  var pkg = JSON.parse(read(path))
+  console.log('v' + pkg.version)
+  process.exit()
+}
+
+var path = resolve(argv.package)
+var pkg = JSON.parse(read(path))
+var deps = pkg.dependencies ? Object.keys(pkg.dependencies) : []
+var devDeps = pkg.devDependencies ? Object.keys(pkg.devDependencies) : []
+
+if (deps.length) {
+  exec('npm i ' + deps.join(' '), { maxBuffer: Infinity, stdio: 'inherit' })
+}
+
+if (devDeps.length) {
+  exec('npm i -D ' + devDeps.join(' '), { maxBuffer: Infinity, stdio: 'inherit' })
```

Comment:
Having two `exec` commands adding things to `node_modules` at the same time is dangerous. They may download dup packages or clash. That's why they were smashed into one command. This should be fixed.

Reply:
this might be the case if I was using the standard async `exec` method, but in this case i used `execSync` since there's no real need for the installation to run asynchronously. if that's difficult to tell at first glance, i might cut down on the dependency renaming for clarity's sake